manifests/params.pp

```diff
@@ -14,7 +15,6 @@
   }
 
   $consul_version           = '0.6.1'
-  $consul_advertise_addr    = '127.0.0.1'
```

Comment:
Nope. We use the node's address as Consul's advertise address (see `$address` in controller/worker). This is old dead code from a previous fix.